packages/wrangler/src/index.tsx

```diff
@@ -880,6 +880,12 @@ export async function main(argv: string[]): Promise<void> {
           initialMode={args.local ? "local" : "remote"}
           jsxFactory={args["jsx-factory"] || envRootObj?.jsx_factory}
           jsxFragment={args["jsx-fragment"] || envRootObj?.jsx_fragment}
+          localProtocol={
+            // The typings are not quite clever enough to handle element accesses, only property accesses,
+            // so we need to cast here.
+            (args["local-protocol"] as "http" | "https") ||
+            config.dev.upstream_protocol
```

Comment:
wait, shouldn't this be 
```suggestion
            config.dev.local_protocol
```

Reply:
Oops!